test/circuit_runner/test_circuit_runner.py

```diff
@@ -0,0 +1,53 @@
+# This code is part of qiskit-runtime.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test the circuit runner program."""
+
+# import sys, os
+# sys.path.insert(0, os.path.join(os.getcwd(), "../../qiskit_runtime"))
+
+from qiskit.providers.aer import AerSimulator
+from qiskit.providers.ibmq.runtime.utils import RuntimeEncoder, RuntimeDecoder
+from qiskit_runtime.circuit_runner import circuit_runner
+from qiskit.result.result import Result
+from qiskit import QuantumCircuit
+from test.fake_user_messenger import FakeUserMessenger
+
+import json
+from unittest import TestCase
+
+class TestCircuitRunner(TestCase):
+    """Test circuit_runner."""
+
+    def setUp(self) -> None:
+        """Test case setup."""
+        N = 6
+        qc = QuantumCircuit(N)
+        qc.x(range(0, N))
+        qc.h(range(0, N))
+        self.qc = qc
+        self.backend = AerSimulator()
+        self.user_messenger = FakeUserMessenger()
+        
+    def test_circuit_runner(self):
+        """Test circuit_runner program."""
+        input = {
+            "metadata": "foo"
+        }
+        serialized_inputs = json.dumps(input, cls=RuntimeEncoder)
+        unserialized_inputs = json.loads(serialized_inputs, cls=RuntimeDecoder)
```

Comment:
Why do you only un/serialize `metadata` and not all parameters (e.g. circuits)? `metadata` also is not a valid input to `circuit_runner`.